zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -748,6 +749,16 @@ private void removeAllAngularObjectInParagraph(String user, String paragraphId)
   }
 
   public void persist(AuthenticationInfo subject) throws IOException {
```

Comment:
Agree that `forcePersist` is a  little confusing. Without reading the code, it is hard to guess what does it mean. 
How about name it as `persist(PersistenceStragtegy strategy)` ?